### PR TITLE
Remove CompareTwoInstances.hh

### PR DESCRIPTION
**Pull request by Teemperor**
This file hasn't been modified in years and also can't be parsed
because it has a stray "/" in it that was supposed to be a line
comment, so I think we can remove it.

### CODE REVIEW

**Comment by Teemperor** [comment]
Part of the work going on regarding the C++ modules migration of CMSSW (tracked as issue https://github.com/cms-sw/cmssw/issues/15248). This PR is not meant to refactor things, but just to make these headers compile.

File: DQM/L1TMonitor/interface/CompareTwoInstances.hh [hunk 1]
```diff
@@ -1,115 +0,0 @@
...
-// class declaration
-//
-// In your class define two collections and their types and input tags
-//   edm::InputTag Coll1_, Coll2_;
-//   CompareTwoInstances<L1GctEmCand, unsigned> a_;
-
-// initialize in the constructor
-//  Coll1_ = iConfig.getUntrackedParameter<edm::InputTag>("Coll1");
-//  Coll2_ = iConfig.getUntrackedParameter<edm::InputTag>("Coll2");
-/
```
Comment (Teemperor) [inline DQM/L1TMonitor/interface/CompareTwoInstances.hh]: Just to point out the stray `/`

**Comment by davidlange6** [comment]
Please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
merge



On 16 Jun 2017, at 17:09, Raphael Isemann <notifications@github.com<mailto:notifications@github.com>> wrote:


This file hasn't been modified in years and also can't be parsed
because it has a stray "/" in it that was supposed to be a line
comment, so I think we can remove it.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/cms-sw/cmssw/pull/19298

Commit Summary

  *   Remove CompareTwoInstances.hh

File Changes

  *   D DQM/L1TMonitor/interface/CompareTwoInstances.hh<https://github.com/cms-sw/cmssw/pull/19298/files#diff-0> (115)

Patch Links:

  *   https://github.com/cms-sw/cmssw/pull/19298.patch
  *   https://github.com/cms-sw/cmssw/pull/19298.diff

**Pull request merged**